src/FixedDataTableContainer.js

```diff
@@ -67,14 +75,23 @@ class FixedDataTableContainer extends React.Component {
   }
 
   render() {
-    return (
-      <FixedDataTable
-        {...this.props}
-        {...this.state}
-        scrollActions={this.scrollActions}
-        columnActions={this.columnActions}
-      />
+    const fdt = (
+        <FixedDataTable
+            {...this.props}
+            {...this.state}
+            scrollActions={this.scrollActions}
+            columnActions={this.columnActions}
+        />
     );
+    // For backward compatibility, by default we render FDT-2 scrollbars
+    if (this.props.defaultScrollbars) {
+      return (
+          <ScrollContainer {...this.props}>
+            {fdt}
+          </ScrollContainer>
+      );
+    }
+    return fdt;
```

Comment:
I'm a bit confused on how the **custom scrollbars** get "props" at this level.

When `defaultScrollbars` is on, we pass `this.props` to the default scrollbars by
```js
if (this.props.defaultScrollbars) {
      return (
          <ScrollContainer {...this.props}>
            {fdt}
          </ScrollContainer>
      );
    }
```

Similarly shouldn't we somehow also pass it to the custom scrollbars? Maybe we should avoid passing of props alltogether at this level, and ensure that `<FixedDataTable />` passes everything that's required through `onScrollBarsUpdate` ?

Reply:
here we are passing props to `ScrollContainer` to render FDT and also pass `touchScrollEnabled` and `isRTL` prop

in case of custom scrollbar people will have access to these props (as they are the one passing them to FDT) and can pass directly to scrollbars if required.